types/testing-library__jest-dom/test/testing-library__jest-dom-global-tests.ts

```diff
@@ -19,6 +19,8 @@ expect(element).toHaveAttribute('attr', 'yes');
 expect(element).toHaveClass();
 expect(element).toHaveClass('cls1');
 expect(element).toHaveClass('cls1', 'cls2', 'cls3', 'cls4');
+expect(element).toHaveDisplayValue('str');
+expect(element).toHaveDisplayValue(['str1', 'str2']);
```

Comment:
we're still missing the tests to check that regex is valid anywhere a string could be valid